eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -31,36 +31,63 @@ stages:
   - stage: Signing
     dependsOn: ${{parameters.DependsOn}}
     jobs:
-      - deployment: SignPackage
-        environment: esrp
+      - job: SignPackage
         timeoutInMinutes: 20
         variables:
           - template: /eng/pipelines/templates/variables/globals.yml
           - template: /eng/pipelines/templates/variables/image.yml
         pool:
-          name: azsdk-pool-mms-ubuntu-2004-general
-          image: azsdk-pool-mms-ubuntu-2004-1espt
-          os: linux
-        strategy:
-          runOnce:
-            deploy:
-              steps:
-                - checkout: none
-                - template: /eng/pipelines/templates/steps/download-credscan-suppressions.yml
-
-                - download: current
-                  displayName: 'Download Artifact: ${{parameters.ArtifactName}}'
-                  artifact: ${{parameters.ArtifactName}}
-
-                - template: tools/java-esrp-signing/java-esrp-signing.yml@azure-sdk-build-tools
-                  parameters:
-                    Artifacts: ${{parameters.Artifacts}}
-                    ArtifactDirectory: $(Pipeline.Workspace)/${{parameters.ArtifactName}}
+          name: azsdk-pool-mms-win-2022-general
```

Comment:
Potential clean-up: We should be able to use our variables from image.yml. Such as `$env:WINDOWSVMIMAGE`